recipe/meta.yaml

```diff
@@ -10,17 +10,18 @@ source:
   sha256: 877d4e080f6ded85d566127c7256de8afae6ab4fdc7ef301f7cc906a8177e3e8
 
 build:
-  skip: True  # [py<35]
-  number: 1
+  noarch: python
+  skip: True
```

Comment:
you need to remove this

Reply:
Commit 823735a fix remove the skip statement ;)